hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestOzoneClientMultipartUploadWithFSO.java

```diff
@@ -311,6 +316,124 @@ public void testUploadPartOverrideWithRatis() throws IOException {
         commitUploadPartInfo.getPartName());
   }
 
+  @Test
+  public void testUploadTwiceWithEC() throws IOException {
+
+    String volumeName = UUID.randomUUID().toString();
+    String bucketName = UUID.randomUUID().toString();
+    String keyName = UUID.randomUUID().toString();
+    StringBuilder sb = new StringBuilder("sample Value");
```

Comment:
Could you create a small private method to create the same data, rather than duplicating the logic twice in this test and the other now one?

Reply:
updated @sodonnel